primap2/_aggregate.py

```diff
@@ -717,112 +802,209 @@ def fill_na_gas_basket_from_contents(
             )
         )
 
-    def aggregate_coordinates(
+    def add_aggregates_coordinates(
```

Comment:
Do you mean `add_aggregated_coordinates` (with a **d** instead of an **s**)? It sounds a bit weird as is.

Reply:
It's named `add_aggregates_coordinates` on purpose as I add aggregates on coordinated and don't aggregate coordinates (which in my mind would be some sort of dimensionality reduction). but I agree the name sounds a bit weird.

Reply:
Ahh, "aggregates" is a noun. Yeah, that makes sense.